charts/trino/values.yaml

```diff
@@ -114,6 +114,70 @@ server:
     #    selectPolicy: Max
     # ```
 
+  # -- Configure [KEDA](https://keda.sh/) for workers.
```

Comment:
We should document this is exclusive with `server.autoscaling`, and if possible, help users make the choice, if they're just starting with autoscaling and don't know either one of those options.

Reply:
This makes totally sense. I will improve the documentation on this.

Reply:
I tried to clarify this in the documentation. I also added a warning in NOTES.txt to indicate that keda would take precedente over hpa in case they are both enabled.

Reply:
I tried to clarify this in the documentation. I also added a warning in NOTES.txt to indicate that keda would take precedente over hpa in case they are both enabled.